lib/selector/tokenizer/tokenizer.go

```diff
@@ -54,7 +54,7 @@ type Token struct {
 }
 
 const (
-	identifierExpr = `[a-zA-Z_./-][a-zA-Z0-9_./-]*`
+	identifierExpr = `[a-zA-Z0-9]?([a-zA-Z0-9_.-]{0,61}[a-zA-Z0-9])*`
```

Comment:
I'll update the regex, as you are probably right and we'll need to see the prefixes as well.  Looking into adding UTs for the regex.